cmd/s2i/main.go

```diff
@@ -171,6 +171,7 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVarP(&(cfg.Ref), "ref", "r", "", "Specify a ref to check-out")
 	buildCmd.Flags().StringVarP(&(cfg.AssembleUser), "assemble-user", "", "", "Specify the user to run assemble with")
 	buildCmd.Flags().StringVarP(&(cfg.ContextDir), "context-dir", "", "", "Specify the sub-directory inside the repository with the application sources")
+	buildCmd.Flags().StringVarP(&(cfg.ExcludeRegExp), "exclude-regex", "", "", "Regular expression string for selecting which files from the application source tree to exclude from the tar stream")
```

Comment:
have renamed queued up;

As to the help update, would it be sufficient to point them to the [doc for the go regexp package](https://golang.org/pkg/regexp/)?

As to how to specify multiple patterns, minimally from looking at [defaul exclusion pattern](https://github.com/openshift/source-to-image/blob/master/pkg/tar/tar.go#L27), the separation parameter (or at least one of the valid ones), is `|`